discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryResult.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * QueryResult.
+ */
+public class QueryResult extends HashMap<String, Object> {
+
+  /**
+   * Gets the id.
+   *
+   * @return the id
+   */
+  public String getId() {
+    return (String) this.get("id");
+  }
+
+  /**
+   * Gets the score.
+   *
+   * @return the score
+   */
+  public Double getScore() {
+    return (Double) this.get("score");
+  }
+
+  /**
+   * Gets the metadata.
+   *
+   * @return the metadata
+   */
+  public Map<String, Object> getMetadata() {
```

Comment:
i'm not sure `metadata` is always present on the return object, whereas `id` and `score` are always present - maybe just remove this?